selfswab/models.py

```diff
@@ -103,18 +103,12 @@ class SelfSwabTest(models.Model):
     RESULT_POSITIVE = "Positive"
     RESULT_NEGATIVE = "Negative"
     RESULT_REJECTED = "Rejected"
-    RESULT_EQV = "Equivocal"
-    RESULT_INCONCLUSIVE = "Inconclusive"
-    RESULT_INDETERMINATE = "Indeterminate"
     RESULT_INVALID = "Invalid"
     RESULT_TYPES = (
         (RESULT_PENDING, "Pending"),
         (RESULT_POSITIVE, "Positive"),
         (RESULT_NEGATIVE, "Negative"),
         (RESULT_REJECTED, "Rejected"),
-        (RESULT_EQV, "Equivocal"),
-        (RESULT_INCONCLUSIVE, "Inconclusive"),
-        (RESULT_INDETERMINATE, "Indeterminate"),
```

Comment:
Do we maybe need a data migration to move all the existing values of these to `Invalid`? Otherwise they'll still be these values in the database for existing entries

Reply:
There is no data yet, we're still waiiting on meditech (this API we are pulling the `result` from) to give us API details, hence no deployment for that yet